### ISSUE TITLE
Shoes Classification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/DL-Simplified#37]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Shoes Classification</br>
:red_circle: **Aim** : Create a model to identify the brands of the shoes.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/ifeanyinneji/nike-adidas-shoes-for-image-classification-dataset</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by joyracheal** [comment]
Full name : Emiaka Joy Racheal
GitHub Profile Link :www.github.com/joyracheal
Email ID : rachealjoy804@gmail.com 
Participant ID (if applicable): no
Approach for this Project : I am new to deep learning this is an opportunity to learn about deep learning and contribute to deep learning open source 
What is your participant role? Social summer of code

**Comment by abhisheks008** [comment]
Issue assigned to you @joyracheal. keep learning 🚀

**Comment by adithya-s-k** [comment]
Can you assign me this Issue

- Full name :  Adithya S Kolavi
- GitHub Profile Link :  https://github.com/adithya-s-k
- Email ID : adithyaskolavi@gmail.com
- Participant ID (if applicable): Not applicable
- Approach for this Project: Using Tensorflow, Keras, CNN , SVM
- What is your participant role? (Mention the Open Source program)
SSOC Participant
issue#37

**Comment by abhisheks008** [comment]
Issue assigned to you @adithya-s-k

**Comment by adithya-s-k** [comment]
I was working on the SVM-based Image classification model and I used pickle to export the model as a **.p** file but the size exceeds 100 MB , so I am not able to push it to GitHub.
Do you know of any way I can solve this problem @abhisheks008

**Comment by abhisheks008** [comment]
Deep learning models are huge in size and GitHub does not support files more than 100MB, we can not do anything about it. You don't need to put the `.pkl` file, just upload the other necessary files.

@adithya-s-k

**Comment by adithya-s-k** [comment]
@abhisheks008 I have created 2 models using SVM and CNN and I am getting an accuracy of 75 to 80 percent.
I am still a beginner and did my best to optimize the model.
Is this accuracy enough and can I create a pull request?
![image](https://user-images.githubusercontent.com/27956426/185991507-05279ab3-632c-4065-85f2-d9e4e8bc11ef.png)

**Comment by abhisheks008** [comment]
Yeah it's okay @adithya-s-k

**Comment by adithya-s-k** [comment]
@abhisheks008  please look into this PR - https://github.com/abhisheks008/DL-Simplified/pull/102
